loadgen/kitchen_sink_executor_test.go

```diff
@@ -511,6 +511,62 @@ func TestKitchenSink(t *testing.T) {
 			},
 			historyMatcher: PartialHistoryMatcher(`WorkflowExecutionSignaled`),
 		},
+		{
+			name: "ClientSequence/Signal/Deduplication",
+			testInput: &TestInput{
+				WorkflowInput: &WorkflowInput{
+					ExpectedSignalCount: 10,
+					InitialActions: ListActionSet(
+						NewAwaitWorkflowStateAction("signals_complete", "true"),
+					),
+				},
+				ClientSequence: &ClientSequence{
+					ActionSets: []*ClientActionSet{
+						{
+							Actions: NewSignalActionsWithIDs(1, 2, 3, 4, 5, 6, 7, 8, 9, 10),
```

Comment:
Probably best to just give a range here instead of having to provide each individual signal ID.